src/silo/preprocessing/sequence_info.test.cpp

```diff
@@ -0,0 +1,61 @@
+#include "silo/preprocessing/sequence_info.h"
+
+#include <gtest/gtest.h>
+#include <duckdb.hpp>
+
+#include "silo/preprocessing/preprocessing_exception.h"
+#include "silo/storage/reference_genomes.h"
+
+using silo::ReferenceGenomes;
+using silo::preprocessing::SequenceInfo;
+
+TEST(SequenceInfo, validatesSuccessfulOnCorrectFile) {
+   ReferenceGenomes reference_genomes =
+      ReferenceGenomes::readFromFile("testBaseData/exampleDataset2/reference_genomes.json");
+   SequenceInfo sequence_info(reference_genomes);
+
+   duckdb::DuckDB duckdb;
+   duckdb::Connection connection(duckdb);
+   ASSERT_NO_THROW(
+      sequence_info.validate(connection, "testBaseData/exampleDataset2/sample.ndjson.zst")
```

Comment:
Yes, we certainly can. I also have additional testing ndjson files with only 1-3 uncompressed genomes. Otherwise, the file is too big for my taste. We already talked about the better testing framework, which we can now maybe tackle in December. I added an issue #226